### DISCUSSION TITLE
Failed to build app with entrypoint test_bundle.dart for iOS simulator (xcodebuild exited with code 65)

### DISCUSSION THREAD

**Discussion by Piyu-sh-arma** [leancodepl/patrol#1412] [Q&A] [answered]
I did set up according to tutorial for IOS. but I run the command 
`patrol test --target integration_test/example_test.dart --release --verbose`

I get the following error for both  physical device or emulator

✗ Failed to build app with entrypoint test_bundle.dart for iOS simulator (xcodebuild exited with code 65) (42.9s)
Error: xcodebuild exited with code 65.

Attaching the details logs [logfile.log](https://github.com/leancodepl/patrol/files/11885989/logfile.log)

 and screen shots :

<img width="1206" alt="Screenshot 2023-06-28 at 12 02 10 AM" src="https://github.com/leancodepl/patrol/assets/31820020/af770872-437c-426d-8f32-8db6be3eb070">
<img width="1210" alt="Screenshot 2023-06-28 at 12 02 39 AM" src="https://github.com/leancodepl/patrol/assets/31820020/8776b0ae-4cb6-4bba-80be-8d5c3e23f812">
<img width="1208" alt="Screenshot 2023-06-28 at 12 03 09 AM" src="https://github.com/leancodepl/patrol/assets/31820020/db89f524-c405-477b-8f42-8b6ea1d9b613">
<img width="929" alt="Screenshot 2023-06-28 at 12 03 49 AM" src="https://github.com/leancodepl/patrol/assets/31820020/d470388c-08e9-4ae2-8bf3-208fb98312ee">
<img width="930" alt="Screenshot 2023-06-28 at 12 04 04 AM" src="https://github.com/leancodepl/patrol/assets/31820020/ca8ca1b1-5a33-4b1e-bb4f-68924c47f1d1">

**Comment by bartekpacia** [comment, marked as answer]
Hi, you've got two problems:

- you don't have code signing:

  ```
  error: No profiles for 'com.patrol.demoapp.xctrunner' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.patrol.demoapp.xctrunner'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild. (in target 'RunnerUITests' from project 'Runner')
  ```

- script in `xcode_backend embed_and_thin` is entered wrong. [See a similar issue from yesterday](https://github.com/leancodepl/patrol/issues/1401) where I provided a solution

**Reply by Piyu-sh-arma** [reply]
Thanks. It helped.
